package.json

```
@@ -13,7 +13,7 @@
   },
   "dependencies": {
     "elasticio-node": "0.0.8",
-    "elasticio-sailor-nodejs": "2.2.2"
+    "elasticio-sailor-nodejs": "2.4.2"
```

2.4.2 is already published
```suggestion
    "elasticio-sailor-nodejs": "2.4.3"
```